.release/security-scan.hcl

```diff
@@ -0,0 +1,13 @@
+container {
+	dependencies = true
+	alpine_secdb = true
+	secrets      = true
+}
+
+binary {
+	secrets      = true
+	go_modules   = true
+	osv          = true
+	oss_index    = true
+	nvd          = true
+}
```

Comment:
I'm curious why these aren't the default settings?

Reply:
This is a really good question that I don't think I quite have the answer to. Let me do some sleuthing and circle back to you on this.